lib/mqtt/command.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Mqtt.Command do
+  @moduledoc """
+  Documentation for `Command` contains helper functions for decoding
```

Comment:
Documentation is misleading, I read them a lot to understand what the module or function does

Reply:
I have a process question about how to fix this. Yes I did cut/paste and put in minimal docs as a start. For me it was enough to explain what it was. I did presume there would be higher level documentation at the readme explaining bigger picture and then this would be more perfunctory (ie you would know the overall objective is parsing and executing OpenC2 commands received over mqtt) so this as is would make sense. But I'm too close to this so I don't see what is misleading. Could you say more so I know what to add? And the process question is "Assuming I want to change something, what is correct way to do that in the middle of a PR?" Do I make another commit and add to PR?